homeassistant/components/yale/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""Support for Yale devices."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import cast
+
+from aiohttp import ClientResponseError
+from yalexs.exceptions import AugustApiAIOHTTPError
```

Comment:
Will this exception remain after everything is done or is it just temporary?

Reply:
Eventually the lib will be split up a bit more. Its going to be iterative